doc/api/cli.md

```diff
@@ -944,7 +945,7 @@ In the `nodeOptions` field, only flags that are allowed in [`NODE_OPTIONS`][] ar
 No-op flags are not supported.
 Not all V8 flags are currently supported.
 
-It is possible to use the [official JSON schema](../node_config_json_schema.json)
+It is possible to use the [official JSON schema](../node-config-schema.json)
```

Comment:
Re #57485 , the `../` points to `https://nodejs.org/node-config-schema.json` when under `nodejs.org/api/`, was this the intention?

Reply:
yes https://nodejs.org/docs/latest/api/cli.html#--experimental-config-fileconfig you can check its working on the website

Reply:
But the link is broken when reading the docs under `nodejs.org/api`, was the intention to have a file at https://nodejs.org/node-config-schema.json that just returns the latest config schema?

Reply:
yes feel free to send a pr

Reply:
Please ping web-infra for these types of changes in the future so we know to support it when its released